docs/installing/advanced_man_install.md

```diff
@@ -40,7 +40,9 @@ This content describes each manual step to install the {{site.data.keyword.edge_
 
 4. Use whichever following section that applies to your type of edge device.
 
-### Installing the agent on Linux (ARM 32-bit, ARM 64-bit, or x86_64) edge devices or virtual machines
+**Note**: If your Linux is not one of the supported distributions and it supports containers, see [Container Agent Installation](https://github.com/open-horizon/anax/blob/master/docs/agent_container_manual_deploy.md) for instructions about how to use the containerized agent.
```

Comment:
We are in the process of migrating the "anax" repo documentation from GitHub proper to the /docs/anax/ path of the documentation web site.  We may need to do a global search-and-replace and at the same time convert it from an external link to an internal link.

Reply:
Agreed. This commit just adds the **Note** on L43 to sync the docs.  What we do to migrate the anax repo docs would be a separate work effort.